actors/builtin/reward/reward_actor.go

```diff
@@ -109,14 +112,20 @@ func (a Actor) AwardBlockReward(rt vmr.Runtime, params *AwardBlockRewardParams)
 	return &adt.EmptyValue{}
 }
 
-func (a Actor) WithdrawReward(rt vmr.Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
+func (a Actor) WithdrawReward(rt vmr.Runtime, minerin *address.Address) *adt.EmptyValue {
 	rt.ValidateImmediateCallerType(builtin.CallerTypesSignable...)
```

Comment:
I concur that the miner address needs to be a parameter, but in that case we probably want to lock down the caller to be the owner (or possibly the owner|worker). Wide open means _anyone_ could cause a miner's reward to be paid out to the owner account, which it's hard to see a legitimate reason for, and could be part of an attempted theft.

So I suggest `ValidateImmediateAddr(owner)` after resolving the owner.

Reply:
I can see many reasons to not want to use your owner key here, I think restricting to owner or worker makes sense